tools/getkeys/index.js

```diff
@@ -1,18 +1,22 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-constant-condition */
+/* eslint-disable no-return-await */
+
 /*!
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
 
-const fs = require("fs");
-const os = require("os");
-const path = require("path");
-const util = require("util");
-const { exec } = require('child_process');
-const msRestAzure = require("ms-rest-azure");
-const keyVault = require("azure-keyvault");
-const rcTools = require("@fluidframework/tool-utils");
+import { appendFile as _appendFile } from "fs";
+import { homedir } from "os";
+import { join, basename, extname } from "path";
+import { promisify } from "util";
+import { exec } from "child_process";
```

Comment:
For the native node libraries let's keep their name as the import, so it's easy to see in the code where they come from. That's how they're usually imported in the rest of the repo. The calls will look like they did before (e.g. `os.homedir()` instead of `homedir()`) but that's ok.
```suggestion
import fs from "fs";
import os from "os";
import path from "path";
import util from "util";
import child_process from "child_process";
```